test/driver.js

```diff
@@ -79,6 +75,21 @@ function writeSVG(svgElement, ctx) {
   });
 }
 
+function writeSVG(svgElement, ctx) {
+  // We need to have UTF-8 encoded XML.
+  const svg_xml = unescape(
+    encodeURIComponent(new XMLSerializer().serializeToString(svgElement))
+  );
+  if (svg_xml.includes("background-image: url(&quot;data:image")) {
+    // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1844414
+    // we load the image two times.
+    return loadImage(svg_xml, null).then(() => {
+      return loadImage(svg_xml, ctx);
+    });
+  }
```

Comment:
Could we perhaps mark the function as `async` and just do the following instead?
```suggestion
  if (svg_xml.includes("background-image: url(&quot;data:image")) {
    // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1844414
    // we load the image two times.
    await loadImage(svg_xml, null);
  }
```